### PR TITLE
Add experimental non-documented function to simplify a 3D polyline

### PR DESCRIPTION
**Pull request by sloriot**
@lrineau if I put two lines in the `.cmd` does it run twice the exe?

### CODE REVIEW

**Comment by lrineau** [comment]
> @lrineau if I put two lines in the `.cmd` does it run twice the exe?

That is the intended behavior. I hope it still works that way.

**Comment by maxGimeno** [comment]
```
The following files have trailing whitespaces:
Polygon_mesh_processing/test/Polygon_mesh_processing/test_simplify_polylines_pmp.cpp
```

**Comment by sloriot** [comment]
> > @lrineau if I put two lines in the `.cmd` does it run twice the exe?
> 
> That is the intended behavior. I hope it still works that way.

it does not work with ctest, let me check with `cgal_test_with_cmake`

**Comment by sloriot** [comment]
Does not work neither, but I know it has always been like that, AFAIR

**Comment by sloriot** [comment]
@maxGimeno you might want to create a plugin for the demo using the function.

File: Polyhedron/demo/Polyhedron/Plugins/IO/Polylines_io_plugin.cpp [hunk 1]
```diff
@@ -253,6 +268,29 @@ void Polyhedron_demo_polylines_io_plugin::split()
   }
 }
 
+void Polyhedron_demo_polylines_io_plugin::simplify()
+{
+  Scene_polylines_item* item = qobject_cast<Scene_polylines_item*>(scene->item(scene->mainSelectionIndex()));
+  bool ok;
+  double err = QInputDialog::getDouble(mw, "Brechet Distance", "Enter the approximation error:", pow(0.01*item->diagonalBbox(),2),0,999,8,&ok);
```
Comment (sloriot) [inline Polyhedron/demo/Polyhedron/Plugins/IO/Polylines_io_plugin.cpp:275]: Who is Brechet? Squared Distance or do the square internally?

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.2-Ic-41.shtml

**Pull request merged**